### PR TITLE
Write conflicts tag into primary xml

### LINKED ISSUE #508: Write conflicts list into primary xml
Let's add list of conflicts into primary xml.

### PR DESCRIPTION
**Pull request by olenagerasimova**
Closes #508 
Added conflicts list into primary xml and corresponding tests.

### CODE REVIEW

File: src/main/java/com/artipie/rpm/meta/XmlEventPrimary.java [hunk 1]
```diff
@@ -287,6 +288,49 @@ private static void addObsoletes(final XMLEventWriter writer, final HeaderTags t
...
+            events.createStartElement(XmlEventPrimary.PRFX, XmlEventPrimary.NS_URL, "conflicts")
+        );
+        final List<Optional<String>> flags = tags.conflictsFlags();
+        final List<HeaderTags.Version> versions = tags.conflictsVer();
+        final Set<String> items = new HashSet<>(names.size());
+        for (int ind = 0; ind < names.size(); ind = ind + 1) {
+            final String concat = names.get(ind).concat(versions.get(ind).toString())
+                .concat(flags.get(ind).orElse(""));
+            if (items.contains(concat)) {
+                continue;
```
Comment (g4s8) [inline src/main/java/com/artipie/rpm/meta/XmlEventPrimary.java:315]: Is it possible to cover this logic in test too?

**Review by g4s8 — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, please check one comment

**Comment by olenagerasimova** [comment]
@g4s8 thanks, added test

**Review by g4s8 — APPROVED** [review]
@olenagerasimova thanks

**Pull request merged**